app/src/main/java/dk/jens/backup/AppInfo.java

```diff
@@ -0,0 +1,89 @@
+package dk.jens.backup;
```

Comment:
Alright, I just come from a world where saying in a comment where the thing comes from (like at l. 3) is enough and makes code exploration much more usable but if that's the way things are usually done in Java...